### DISCUSSION TITLE
Negspacy for noun phrase

### DISCUSSION THREAD

**Discussion by iamyihwa** [explosion/spaCy#12083] [Help: Other Questions] [answered]
## How to reproduce the behaviour

<img width="600" alt="image" src="https://user-images.githubusercontent.com/12494156/211301031-410c2e16-c311-4e1b-bdca-9dde5f287ec3.png">

## Your Environment

<img width="500" alt="image" src="https://user-images.githubusercontent.com/12494156/211300518-177c7c30-4f8e-4ae4-9c3d-cfa1a5caa527.png">

In the example of [negspacy](https://pypi.org/project/negspacy/) only entities detected with NER were negated. 
I have noticed that there is property negex also for chunks. 
However it doesn't seem to be working correctly. 
(In the above example of 'I didn't buy hamburger', correct output should be  hamburger True. 

I wonder if this is the right way to use it, or it is a feature currently not developed.

**Comment by shadeMe** [comment, marked as answer]
This seems like an issue with negspacy itself as opposed to the core spaCy library. We only provide support for the latter, so we would suggest that you contact the developers of negspacy to get help with this issue.
